content/molecules.html

```diff
@@ -162,6 +164,20 @@ <h2 class=st-title>Socials</h2>
         </li>
       </ul>
     </article>
+    
+    <article class=st-article>
+      <h2 class=st-title>User profile header</h2>
+      
+      <div class=profile-user>
+        <h1 class=username>User name</h1>
+        <p class=about>Some about content to define user</p>
+        <p>User is currently around
+          <a
```

Comment:
a default text in the `<a>` would be great.
The app in prod has "the world" as default